config/azure/build.yaml

```diff
@@ -53,18 +58,28 @@ jobs:
        DISPLAY: :99.0
 
    - script: |
-       echo BUILD_TYPE=${{ parameters.buildType }}
+       BUILD_TYPE=${{ parameters.buildType }}
```

Comment:
What's this doing? We seem to have made a global `BUILD_TYPE` variable above anyway (line 27), and below we're using `parameters.buildType` anyway.

Reply:
It was just so you could look at the output and see. For informational purposes only.

Reply:
I'm confused. I thought that's what the old line was doing. The new line appears to be setting a variable in bash...

Reply:
Ohhh I see whats happened, when I removed my 'testing speed it up by echoing stuff instead of doing it' workaround, I inadvertently removed the one actual `echo` that was _meant_ to be there.... doh!

Reply:
Ah! That makes sense! Could you fix that up real quick and then merge?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3242/commits/427834735fa551636495a8a0624ef00eb2b9e273

Reply:
Merged as requested